src/Navbar/index.js

```diff
@@ -9,7 +9,11 @@ export default () => (
         {!window.email ?
           <li className="navigation-item"><Link to='/login' className="navigation-link">Login</Link></li>
           :
-          <li className="navigation-item"><Link to='/logout' className="navigation-link">Logout</Link></li>
+          <li className="navigation-item">
```

Comment:
Each item is supposed to be a different "navigation-item" (especially when we hover over it and highlight. As react can only have one element in an conditional statement, bring the `<ul clssname...` from up into the condition values.